validator/remote/src/main/java/tech/pegasys/teku/validator/remote/typedef/handlers/SendSignedBlockRequest.java

```diff
@@ -47,7 +51,7 @@ public SendSignedBlockResult sendSignedBlock(SignedBeaconBlock signedBeaconBlock
             ? SEND_SIGNED_BLINDED_BLOCK
             : SEND_SIGNED_BLOCK;
 
-    return preferSszBlockEncoding
+    return preferSszBlockEncoding && !unsupportedMediaType
```

Comment:
I think its probably simpler to handle one request. but if we do want to go this way...

preferSszEncoding was the callers way of defining that the cli is passed in. 

We don't need 2 flags, we just need to know if we're going to bother to try SSZ. so when unsupportedMediaType goes to true, we could instead set preferSszEncoding to false, and call the json endpoint.